src/GeositeFramework/js/PluginBase.js

```diff
@@ -40,11 +40,11 @@ define(["dojo/_base/declare",
             // Allow the framework to put a custom print button for this plugin
             hasCustomPrint: false,
 
-            // Show a print preview map for plugin printing
-            usePrintPreviewMap: false,
+            // Show a modal dialog for plugin printing
+            usePrintModal: false,
 
-            // The [width, height] of the print preview map
-            previewMapSize: [500, 400],
+            // The [width, height] of the print modal
+            printModalSize: [500, 400],
```

Comment:
The size was set specifically since the preview map needed them set explicitly.  Given that this will have form content going forward, are the sizes still needed?  I think it would be convenient if the container fit the contents, but it may be more trouble than it's worth.

Reply:
I think this is a good idea, and I created a follow-up issue: https://github.com/CoastalResilienceNetwork/GeositeFramework/issues/1042.